packages/create-cloudflare/src/__tests__/git.test.ts

```diff
@@ -163,11 +162,23 @@ describe("git helpers", () => {
 				["git", "init", "--initial-branch", "main"],
 				expect.any(Object),
 			);
+			expect(ctx.gitRepoAlreadyExisted).toBe(false);
 			expect(ctx.args.git).toBe(true);
```

Comment:
By the end of `offerGit()` we should have decided whether `git` is required but also whether `gitRepoAlreadyExisted` is true. This is then inherited/used in the `gitCommit()` function later.